packages/client/src/app/shared/global-methods.ts

```diff
@@ -0,0 +1,16 @@
+import { GlobalConstants } from './global-constants';
+
+export class GlobalMethods {
```

Comment:
Two things:

First:
Personal opinion: I'm not a fan of "Util", "Helper", etc rollup files. Names should be descriptive. In this case, I think something like `titles.ts` for the filename and `Titles` for the class name would be more informative here.

Second:
These functions don't actually need to live in a class--the more idiomatic way to define them would be as naked functions, e.g.
```typescript
export function setDefaultTitle(): void {
    // etc
} 
```

If you want the benefit of things being "inside" something else, you can use a namespace, instead!

```typescript
namespace Titles {
  export function setDefaultTitle(): void {
    // etc
  }
}
```

...and just do `import Titles` elsewhere.